themes/src/main/resources/theme/base/login/register.ftl

```diff
@@ -104,6 +104,18 @@
                             </span>
                         </#if>
                     </div>
+                    <div class="${properties.kcInputWrapperClass!}" id="password-entropy" data-password-input="password" style="display: none;">
+                        <div class="progress" style="margin-bottom: 0; margin-top: 1px;">
+                            <div class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">
```

Comment:
Nice, thanks for taking accessibility into account. I am not quite sure that a progress bar is the correct role though, as it is more intended for reporting progress on long-running tasks.

Perhaps a more suitable role for this one would be the [`status` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role), which is a live region that uses `polite`, and thus does not interrupt the user.

```suggestion
                            <div class="progress-bar" role="status">
```